src/map.cpp

```diff
@@ -3100,37 +3071,44 @@ bool map::has_nearby_fire( const tripoint &p, int radius )
     return false;
 }
 
-void map::mop_spills( const tripoint &p ) {
+bool map::mop_spills( const tripoint &p )
+{
+    bool retval = false;
     auto items = i_at( p );
     for( auto it = items.begin(); it != items.end(); ) {
```

Comment:
It would be nice to rewrite this loop in a more "modern" manner.
```c++
const auto new_end = std::remove_if( items.begin(), items.end(), []( const item &it ) {
    return it.made_of( LIQUID );
);
bool retval = new_end != items.end();
items.erase( new_end, items.end() );
```